src/arr/compiler/cli-repl.js

```diff
@@ -20,7 +20,9 @@
       }
       var restartInteractions = runtime.getField(pyretRepl, "restart-interactions");
       return runtime.safeCall(function() {
-          return restartInteractions.app("");
+          // need to convert definitions to JS string?
```

Comment:
This is fine; Pyret strings are JS strings, so this will work without additional changes.

Reply:
Yes, I know it works for now, but I was wondering if I was supposed to wrap it in a conversion function anyway--the docs say something about possible making Pyret strings more complex in the future.